pyteal/ast/addr.py

```diff
@@ -0,0 +1,24 @@
+from ..types import TealType, valid_address
+from ..ir import TealOp, Op
+from .leafexpr import LeafExpr
+
+class Addr(LeafExpr):
+    """An expression that represents an Algorand address."""
+
+    def __init__(self, address: str) -> None:
+        """Create a new Addr expression.
+
+        Args:
+            address: A string containing a valid base32 Algorand address
+        """
+        valid_address(address)
+        self.address = address
+
+    def __teal__(self):
+        return [TealOp(Op.addr, self.address)]
```

Comment:
not sure if it matter and just fyi, `addr` is assembler-level syntax sugar

Reply:
`addr` takes in a base32 Algorand address with checksum, so it's useful to have.